account_financial_report_qweb/report/abstract_report_xlsx.py

```diff
@@ -3,6 +3,8 @@
 # Copyright 2016 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
+from cStringIO import StringIO
+import xlsxwriter
```

Comment:
Hi @pedrobaeza 

I have added the `cStringIO` and `xlsxwriter` dependencies because I have override the `create_xlslx_report` method for "Customization of xlsxwriter options to limit usage of memory for xlsx files generation".

See lines 38 to 51 of this file.